config/rbac/monitoring/rbac.yaml

```diff
@@ -0,0 +1,57 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: local-storage-monitor
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
I see a inconsistency here -  this rbac grants '*/*' permissions in `monitoring.coreos.com` group, but no such equivalent rule exists in CSV (or rbac that was copied in manifests directory). So either this rule is too broad or rule in CSV is too narrow.

Reply:
You are right. This rule is too broad.  Servicemonitors are being created by LSO operator that use `local-storage-operator` service account. We have already grant rbac permissions for `monitoring.coreos.com` group to `local-storage-operator` service account in the CSV -  [here](https://github.com/openshift/local-storage-operator/blob/a3fd1c5c74da1ad40d708a563c0cc8e00fe16135/config/manifests/4.9/local-storage-operator.v4.9.0.clusterserviceversion.yaml#L209-L228 ) and [here](https://github.com/openshift/local-storage-operator/blob/a3fd1c5c74da1ad40d708a563c0cc8e00fe16135/opm-bundle/manifests/local-storage-operator.clusterserviceversion.yaml#L189-L209).  

So removing these permissions for `local-storage-admin` as its not required.